consumers/src/unit-test/java/uk/gov/moj/sdt/cmc/consumers/client/converter/XmlToObjectTest.java

```diff
@@ -56,4 +59,27 @@ void shouldConvertBreathingSpaceRequestAndConvertToJson() throws IOException {
         assertNotNull(jsonString);
         assertEquals(JSON_RESPONSE, jsonString);
     }
+
+    @Test
+    void shouldConvertClaimStatusUpdateRequestToString() throws IOException {
+        String xmlContent = readXmlAsString(CLAIM_STATUS_UPDATE);
+        String request = xmlToObject.convertXmlToJson(xmlContent);
+        assertNotNull(request);
```

Comment:
Add code to check that the content of the request is what is expected

Reply:
done